sysid-application/src/main/native/cpp/view/AnalyzerPlot.cpp

```diff
@@ -327,6 +330,7 @@ void AnalyzerPlot::SetData(const Storage& rawData, const Storage& filteredData,
   // represents the prediction at the timestep, and N represents the number of
   // points
   m_RMSE = std::sqrt(simSquaredErrorSum / timeSeriesPoints);
+  m_RSquared = 1 - m_RMSE / std::sqrt(squaredVariationSum / timeSeriesPoints);
```

Comment:
This sets m_Rsquared to NaN if squaredVariationSum == 0.0 instead of the expected value of 1.

Reply:
This never actually occurs (it would require the data to be uniformly zero), and if it did the value of `1` isn't really helpful.  I don't think it's worth catching.